console-frontend/src/components/hydra-console/HydraConsole.jsx

```diff
@@ -276,7 +291,54 @@ class HydraConsole extends React.Component {
         })
     }
 
+    setResourceID(name, value) {
+        this.getURL = true;
+
+        console.log(name +" "+ value)
+        let resourcesIDs = Object.assign({}, this.state.resourcesIDs);
+        resourcesIDs[name]['ResourceID'] = value.split('/').pop();
 
+        localStorage.setItem('resourceIDs', JSON.stringify(resourcesIDs))
+        
+        this.setState({
+            resourcesIDs: resourcesIDs
+        })
+    }
+
```

Comment:
Please add proper comments/docstrings to explain what each function does

Reply:
It would be great if you can do this for all the functions not just for the functions you wrote. :smile:

Reply:
Sure thing to do, my plan was to add them to every function once it was reviewed! Thank you